geometry_msgs/PoseStamped estCrntArCamPose
geometry_msgs/PoseStamped crntArCamPose
geometry_msgs/PoseStamped estCrntArOdomPose
geometry_msgs/PoseStamped crntArOdomPose
geometry_msgs/PoseStamped estCrntCamOdomPose
geometry_msgs/PoseStamped crntCamOdomPose
